ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml

```diff
@@ -7671,15 +8193,42 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Unverified</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R2210001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.44</ns1:Doc_Sect>
+		<ns1:Description>[In Timezone] Protocol version 16.1 supports this [Timezone] element.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
+	</ns1:Requirement>
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASCAL_R448</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.2.45</ns1:Doc_Sect>
-		<ns1:Description>[In Type] The Type element is a required child element of the Recurrence element (section 2.2.2.35) that specifies the type of the recurrence.</ns1:Description>
+		<ns1:Description>[In Type] The Type element is a required child element of the Recurrence element (section 2.2.2.35) in protocol version 2.5, 12.0, 12.1, 14.4, and 14.1; it is an optional child element of the Recurrence element in protocol versions 16.0 and 16.1.</ns1:Description>
```

Comment:
remove "it is an optional child element of the Recurrence element in protocol versions 16.0 and 16.1" from this RS